src/pages/PaymentCancel/PaymentCancel.tsx

```diff
@@ -1,13 +1,21 @@
 import React from "react";
-import { Button } from 'react-bootstrap';
+import { Button, Col, Container, Row } from 'react-bootstrap';
 import { Link } from 'react-router-dom';
+import './PaymentCancel.css';
 
 function Cancel() {
     return (
-        <div>
-            <h1>Your canceled the payment process</h1>
-            <h2>If you encountered a problem, feel free to contact us via the <Link to="/feedback">Feedback site</Link></h2>
-            <Link to="/"><Button>Back to front page</Button></Link>
+        <div className="payment-cancel">
+            <Container>
+                <Row>
+                    <Col>
+                        <h1 style={{color: "firebrick"}}>Your canceled the payment process</h1>
+                        <p>It seems you decided to not buy premium. We are sorry to hear that. Maybe you will change your mind in the future :)</p>
+                        <p>If you encountered a problem, feel free to contact us via the <Link to="/feedback">Feedback site</Link></p>
```

Comment:
```suggestion
                        <p>If you encountered a problem, please contact us via the <Link to="/feedback">Feedback site</Link> we are happy to help.</p>
```
Does the feedback site have a link to the discord server?